account-decoder/src/parse_token.rs

```diff
@@ -19,16 +15,13 @@ use {
     },
     std::str::FromStr,
 };
-
-// Returns all known SPL Token program ids
-pub fn spl_token_ids() -> Vec<Pubkey> {
-    vec![spl_token::id(), spl_token_2022::id()]
-}
-
-// Check if the provided program id as a known SPL Token program id
-pub fn is_known_spl_token_id(program_id: &Pubkey) -> bool {
-    *program_id == spl_token::id() || *program_id == spl_token_2022::id()
-}
+pub use {
+    solana_account_decoder_client_types::token::{
+        real_number_string, real_number_string_trimmed, TokenAccountType, UiAccountState, UiMint,
+        UiMultisig, UiTokenAccount, UiTokenAmount,
+    },
+    spl_generic_token::{is_known_spl_token_id, spl_token_ids},
+};
```

Comment:
we `pub use` these to avoid a breaking change because `solana-account-decoder` is a crate that (unlike `solana-inline-spl`) is very reasonable for downstream projects to include